package.json

```diff
@@ -21,14 +21,21 @@
   },
   "homepage": "https://github.com/northcoders/be-nc-news#readme",
   "devDependencies": {
+    "husky": "^7.0.0",
     "jest": "^27.5.1",
     "jest-extended": "^2.0.0",
     "pg-format": "^1.0.4",
-    "husky": "^7.0.0"
+    "supertest": "^6.3.0"
   },
   "dependencies": {
```

Comment:
have a think about which packages you do and don't need in final deployment. This is. bit bloated here